index.css

```diff
@@ -24,9 +24,7 @@ body{
     margin: 8px 0;
 }
 
-#survey-form{
-    width: 50%;
-    
+#survey-form{ width: 50%;
     margin: 0 auto;
```

Comment:
I noticed in the css file there is an error in the line 27, for more information please reade this article for best practice HTML&CSS https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md